src/methods.jl

```diff
@@ -56,10 +62,12 @@ subtract_bias(frame::AbstractArray, bias_frame::AbstractArray) = subtract_bias!(
 
 
 """
-    subtract_overscan!(frame::AbstractArray, idxs; dims = axes_min_length(idxs))
+    subtract_overscan!(frame, idxs; dims = axes_min_length(idxs))
 
 In-place version of [`subtract_overscan`](@ref)
 
+`frame` cannot be a string for mutating version.
```

Comment:
Similar to above, to ME it makes sense that you can't mutate on a string, so it shouldn't need an explanation. However, this requires knowledge of the idiosyncrasies of `!` methods, so from that viewpoint its not a terrible idea to explain it. 

If you do explain it, you'll need to be more explicit. The first thing I would saw if I read this is "why?". Unfortunately, this means you'll be copying the same explanation to all the docstrings for consistency.

So I think either remove this or put an extra line explaining why- up to you!

Reply:
Let's remove this, explaining why strings are not supported would make the docs quite verbose, which may intimidate the user :laughing: